akka-docs/src/main/paradox/typed/routers.md

```diff
@@ -57,10 +57,15 @@ The routees, however, are spawned by the router.
 Therefore, the `PoolRouter` has a property to configure the `Props` of its routees:
 
 Scala
-:  @@snip [RouterSpec.scala](/akka-actor-typed-tests/src/test/scala/docs/akka/typed/RouterSpec.scala) { #pool-dispatcher }
+:  @@snip [RouterSpec.scala](/akka-actor-typed-tests/src/test/scala/docs/akka/typed/RouterSpec.scala) { #broadcast }
 
 Java
-:  @@snip [RouterTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/RouterTest.java) { #pool-dispatcher }
+:  @@snip [RouterTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/RouterTest.java) { #broadcast }
```

Comment:
This sample should stay `pool-dispatcher` and new snippets added below

Reply:
sorry, (lack of) copy paste mistake